### PR TITLE
Document optional artifacts

### LINKED ISSUE #160: Document the available Tessera artifacts
Document the artefacts that are available to supplement a distribution:
See: https://docs.tessera.consensys.net/en/latest/HowTo/Get-started/Install/Distribution/ 

For info about the artifacts see:
https://github.com/ConsenSys/tessera/tree/tessera-22.1.1#optional-artefact

### PR DESCRIPTION
**Pull request by rolandtyler**
# Pull request checklist

Use the following template to make sure your PR fits the Tessera documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-locally).

## Describe the change
Documents the available optional artifacts.

## Issue fixed
Fixes #160 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://consensys.net/docs/doctools/en/latest/preview/old-system/#preview-on-read-the-docs)
  and added a [preview link](#preview).

## Preview
https://pegasys-tessera--164.com.readthedocs.build/en/164/HowTo/Get-started/Install/Distribution/#available-optional-artifacts

### CODE REVIEW

File: docs/HowTo/Get-started/Install/Distribution.md [hunk 1]
```diff
@@ -68,3 +68,20 @@ Copy the contents of the artifact's `/lib` into the distribution `/lib` (resolve
...
+
+#### Available optional artifacts
+
+The following artifacts can be added to a distribution to provide additional functionality.
+
+##### Key Vaults
+
+- [Azure](https://github.com/ConsenSys/tessera/tree/tessera-22.1.1/key-vault/azure-key-vault): Adds support for key pairs stored in Azure Key Vault.
+- [AWS](https://github.com/ConsenSys/tessera/tree/tessera-22.1.1/key-vault/aws-key-vault): Adds support for key pairs stored in AWS Secret Store.
+- [HashiCorp](https://github.com/ConsenSys/tessera/tree/tessera-22.1.1/key-vault/hashicorp-key-vault): Adds support for key pairs stored in HashiCorp Vault.
```
Comment (macfarla) [inline docs/HowTo/Get-started/Install/Distribution.md:80]: is there a reason we're linking to a specific version? can we make the link generic so we don't have to update it?

File: docs/HowTo/Get-started/Install/Distribution.md [hunk 2]
```diff
@@ -68,3 +68,20 @@ Copy the contents of the artifact's `/lib` into the distribution `/lib` (resolve
...
  cp -a aws-key-vault-[version]/lib/. tessera-[version]/lib/
 ```
+
+#### Available optional artifacts
+
+The following artifacts can be added to a distribution to provide additional functionality.
+
+##### Key Vaults
+
+- [Azure](https://github.com/ConsenSys/tessera/tree/master/key-vault/azure-key-vault): Adds support for key pairs stored in Azure Key Vault.
```
Comment (macfarla) [inline docs/HowTo/Get-started/Install/Distribution.md:78]: These aren't links to artifacts. To get the artifacts you have to go to dockerhub eg https://hub.docker.com/r/quorumengineering/tessera/tags?page=1&name=aws

Comment (rolandtyler) [reply docs/HowTo/Get-started/Install/Distribution.md:78]: I think we don't want to link to dockerhhub here. This page is for installing by binary distribution. The section just above this is how to [add the artifacts](https://github.com/ConsenSys/doc.tessera/blob/master/docs/HowTo/Get-started/Install/Distribution.md#add-tessera-artifacts) from the .jar files. This was to list the artifacts (and their repo folders), matching the info on the [Tessera readme](https://github.com/ConsenSys/tessera/tree/tessera-22.1.1#optional-artefact).

**Comment by macfarla** [comment]
stale

**Pull request closed without merging**